rfcs/48-diverse-defaults.md

```diff
@@ -0,0 +1,58 @@
+# Feature Name: Different plugin groups to support different use cases (diverse-defaults for short)
+
+## Summary
+
+There are many different categories of Bevy applications. Each use case has distinct needs, and so we should provide a collection of `PluginGroups` to give our users good defaults.
+
+## Motivation
+
+Bevy would benefit from an improved 'getting started' user story for non-minimal bevy applications. This is particularly helpful for beginners, but would also be a nice ergonomics/UX win for experienced bevy devs to not have to configure every minute detail manually when setting up a new project that is in one of the above outlined application domains.
+
+## User-facing explanation
+
+Bevy now provides you a number of convenient `PluginGroups` to help you get your application started with sensible defaults for your use case. The current list is:
+
+- `MinimalPlugins`: The very basic internal structure required to ensure things like `Time` function properly.
+- `GamePlugins`: Suitable for making both 2D and 3D real-time games with input handling, audio and rendering (note: previously `DefaultPlugins`).
+- `2dGamePlugins`: Like `GamePlugins`, but limited to the systems and resources needed in a 2D game.
+- `TuiPlugins`: Suitable for making TUI applications.
+- `ApplicationPlugins`: Suitable for making relatively static non-game applications.
+- `SimulationPlugins`: Plugins that make writing scientific simulations easier like enhanced determinism and ordering guarantees.
+
+## Implementation strategy
+
+Each of the above outlined `Plugins` may have some combination of the following divergent platform configuration needs:
+
+- desktop
+ - native
+ - web
+- tablet
+ - ios native
+ - android native
+ - web
+- phone
+ - ios native
+ - android native
+ - web
+
+This feature also touches on not-yet-implemented but planned bevy features that will have different implementations per use case.
+
+For example, using making low-power consumption web and mobile applications,  desktop productivity apps and high performance games will all have very different scheduling needs.
+
+2D vs 3D will have different built-in bevy selection methods/mechanisms and different cameras.
+
+Building for touch vs mouse vs gamepad or other midi/advanced input controllers different use cases with very different requirements.
+
+## Drawbacks
+
+The primary drawback I can see is the need for Bevy to maintain the multiple `PluginGroups`, however, I foresee the maintenance burden of this to be exceedingly minimal.
+
+## Rationale and alternatives
```

Comment:
I think you should discuss the pros and cons of plugin groups vs. feature groups in this section.

Reply:
How would you define `FeatureGroups`?

Reply:
Rust features can enable other features, allowing users to toggle them on and off as needed. We can also 

See: https://doc.rust-lang.org/cargo/reference/features.html for a basic introduction.
https://docs.rs/cfg_feature_groups/latest/cfg_feature_groups/ or the like may be helpful to improve ergonomics.